### DISCUSSION TITLE
How to line up Input fields vertically without overlapping

### DISCUSSION THREAD

**Discussion by akhran** [tailwindlabs/tailwindcss#1840] [Help]
I have 3 input fields line up vertically. 

1) The bottom border of the first and second input boxes are missing when they are in focus.
2) The bottom border of the first and second input fields seems to be thicker.
3) Can the shadow-outline color be changed?
4) Is there a better class to use to have a border around the input box besides shadow-outline?

Thank you.

```
<form class='flex flex-col items-center justify-center h-screen' action="/" method="post">
  <input class='border-gray-400 border-2 focus:shadow-outline' type="text" name="first" id="first" placeholder="First input">
  <input class='border-gray-400 border-2 focus:shadow-outline' type="text" name="second" id="second" placeholder="Second input">
  <input class='border-gray-400 border-2 focus:shadow-outline' type="text" name="third" id="third" placeholder="Third input">
</form>
```
https://codesandbox.io/s/shy-lake-lue6i?file=/index.html

**Comment by robbyrice** [comment]
In regards to your first two points, because there is no margin between the input fields they are right up against one another and what you see as a thick single border on the bottom of the first and second input fields is in fact two separate borders (the bottom border of one input and the top border of the next input). To solve this you can add some spacing (margin) between the inputs by adding the `spacing-y-4` (you can change this to as much or as little as you want) class to the `form` element, or by adding top and bottom margin directly to the input fields. This will also solve the missing bottom border for the first two input fields.

Concerning the shadow-outline color, I haven't really looked into it but I think you can change it.

Concerning a better class to do the outline, out of the box I don't believe there is a better way to do it.

Hope this helps!

**Comment by akhran** [comment]
Thank you for your reply.

I could insert inline style for "margin-bottom: -2px" so that the border overlap each other and it will appear as a single border. "-space-y-1" seems to be the closest match to margin-bottom: -2px, but the bottom of letters such as 'p' is touching the bottom border. 

The bottom borders of the first and second input element are still missing when in focus with '-space-y-1'. Only the last input field (in focus) has the complete blue border.

Appreciate your help on this.
